examples/1_intro_to_genet.ipynb

```diff
@@ -14,7 +14,7 @@
     "\n",
```

Comment:
Below it looks like you're adding section numbers to the notebooks, I think you missed this one